easybuild/easyconfigs/p/PSI4/PSI4-1.2.1_fix_cmake_release.patch

```diff
@@ -0,0 +1,103 @@
++++ psi4-1.2.1/cmake/autocmake_safeguards.cmake	2018-10-30 16:18:26.650251435 +0100
+@@ -19,8 +19,8 @@
+ string(TOLOWER "${CMAKE_BUILD_TYPE}" cmake_build_type_tolower)
+ string(TOUPPER "${CMAKE_BUILD_TYPE}" cmake_build_type_toupper)
+ 
+-if(NOT cmake_build_type_tolower STREQUAL "debug" AND
+-   NOT cmake_build_type_tolower STREQUAL "release" AND
+-   NOT cmake_build_type_tolower STREQUAL "relwithdebinfo")
+-    message(FATAL_ERROR "Unknown build type \"${CMAKE_BUILD_TYPE}\". Allowed values are Debug, Release, RelWithDebInfo (case-insensitive).")
+-endif()
++#if(NOT cmake_build_type_tolower STREQUAL "debug" AND
++#   NOT cmake_build_type_tolower STREQUAL "release" AND
++#   NOT cmake_build_type_tolower STREQUAL "relwithdebinfo")
++#    message(FATAL_ERROR "Unknown build type \"${CMAKE_BUILD_TYPE}\". Allowed values are Debug, Release, RelWithDebInfo (case-insensitive).")
++#endif()
+diff --new-file -ru psi4-1.2.1.orig/external/upstream/gau2grid/CMakeLists.txt psi4-1.2.1/external/upstream/gau2grid/CMakeLists.txt
+--- psi4-1.2.1.orig/external/upstream/gau2grid/CMakeLists.txt	2018-07-25 04:11:25.000000000 +0200
++++ psi4-1.2.1/external/upstream/gau2grid/CMakeLists.txt	2018-10-30 16:36:23.558755892 +0100
+@@ -16,6 +16,7 @@
+         GIT_REPOSITORY https://github.com/dgasmith/gau2grid
+         GIT_TAG v1.1.0
+         UPDATE_COMMAND ""
++        PATCH_COMMAND  sed -e "s/debug/none/" -i cmake/autocmake_safeguards.cmake
+         CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}
+                    -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+                    -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+diff --new-file -ru psi4-1.2.1.orig/external/upstream/libint/CMakeLists.txt psi4-1.2.1/external/upstream/libint/CMakeLists.txt
+--- psi4-1.2.1.orig/external/upstream/libint/CMakeLists.txt	2018-07-25 04:11:25.000000000 +0200
++++ psi4-1.2.1/external/upstream/libint/CMakeLists.txt	2018-10-30 16:40:02.623920471 +0100
+@@ -15,6 +15,7 @@
+         GIT_REPOSITORY https://github.com/evaleev/libint
+         GIT_TAG 024738c  # v1.2.1+2 release-1-2-1
+         UPDATE_COMMAND ""
++        PATCH_COMMAND  sed -e "s/debug/none/" -i cmake/autocmake_safeguards.cmake
+         CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}
+                    -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+                    -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+diff --new-file -ru psi4-1.2.1.orig/external/upstream/libxc/CMakeLists.txt psi4-1.2.1/external/upstream/libxc/CMakeLists.txt
+--- psi4-1.2.1.orig/external/upstream/libxc/CMakeLists.txt	2018-07-25 04:11:25.000000000 +0200
++++ psi4-1.2.1/external/upstream/libxc/CMakeLists.txt	2018-10-30 16:29:37.513906489 +0100
+@@ -11,10 +11,12 @@
+ 
+     include(ExternalProject)
+     message(STATUS "Suitable Libxc could not be located, ${Magenta}Building Libxc${ColourReset} instead.")
++    string(REPLACE "-xHost" "" CMAKE_C_FLAGS_Libxc "${CMAKE_C_FLAGS} -O1 ")
```

Comment:
I don't think this is required anymore with `intel/2018b`, `libxc` tests pass just fine even when `-xHost` is used (see also discussion in #6593)